package com.enterpriseproject.film.converters;

public class ConverterException extends RuntimeException {

    private static final long serialVersionUID = 1L;

   public ConverterException() {
   }

   public ConverterException(String message) {
      super(message);
   }

   public ConverterException(String message, Throwable cause) {
      super(message, cause);
   }

   public ConverterException(Throwable cause) {
      super(cause);
   }
}